### DISCUSSION TITLE
How to add autocomplete entries ?

### DISCUSSION THREAD

**Discussion by FrenchGithubUser** [atom/atom#22396] [Q&A]
Hi everyone, I'm trying to add autocomplete entries in atom but I can't find any way about how to do so.
It is actually to add html autocomplete entries, is there a way to modify the html autocompletet atom package or do I have to do something else ?
I also want to mention that I couldn't find any documentation about how to do so, anywhere, even on the readme files of various autocomplete addons....

**Comment by ThatXliner** [comment]
Does [this](https://github.com/atom/autocomplete-plus/wiki/Provider-API) help?

**Reply by FrenchGithubUser** [reply]
Snippets was what I was looking for, but thanks anyways !

**Comment by UziTech** [comment]
Maybe [snippets](https://flight-manual.atom.io/using-atom/sections/snippets/) are what you are looking for?

[more info](https://github.com/atom/snippets)

**Reply by FrenchGithubUser** [reply]
Yes exactly ! Thanks many times !!!!
